DATA/UNIVERSE/SYSTEMS/Bw06/Bw06.ini

```diff
@@ -1796,3 +1796,68 @@ rotate = 60, 0, 0
 Archetype = suprise_ge_csv
 visit = 16
 loadout = SECRET_Syd
+
+;; Mineable Gas Pockets
+
+[Object]
+nickname = mineable_small_organisms_bw06_01

```

Comment:
I am still thinking those should be astroids and not gas pockets. But that discussion can be done later.

Reply:
Asteroids are not gas so no

Reply:
Sorry read it not correct, Is it realy that bad to maxbe later have both?.

Alien Organisms is the same general term like hydrocarbons. It can be different types of matter (Gas, Solid, Fluid). 

Need to check the rumors but i dont believe that there is a saying about hiw this organism is living. Thing is if you fly through the omicrons you see the organisms living in the nebula (particles in the nebulae)